airflow/ui/src/pages/Events/Events.tsx

```diff
@@ -121,13 +131,14 @@ export const Events = () => {
     limit: pagination.pageSize,
     offset: pagination.pageIndex * pagination.pageSize,
     orderBy,
+    runId,
```

Comment:
`taskId` can also be passed to the Events API which when available will filter out events for the task in the task instance's events tab.

```suggestion
    runId,
    taskId,
```